AutomatedTesting/Gem/PythonTests/Atom/tests/hydra_AtomEditorComponentsLevel_DiffuseGlobalIlluminationAdded.py

```diff
@@ -84,17 +81,28 @@ def AtomEditorComponentsLevel_DiffuseGlobalIllumination_AddedToEntity():
         Report.result(Tests.creation_redo,
                       EditorLevelEntity.has_component(AtomComponentProperties.diffuse_global_illumination()))
 
-        # 4. Set Quality Level property to Low
-        diffuse_global_illumination_component.set_component_property_value(
-            AtomComponentProperties.diffuse_global_illumination('Quality Level'), GLOBAL_ILLUMINATION_QUALITY['Low'])
-        quality = diffuse_global_illumination_component.get_component_property_value(
-            AtomComponentProperties.diffuse_global_illumination('Quality Level'))
-        Report.result(Tests.diffuse_global_illumination_quality, quality == GLOBAL_ILLUMINATION_QUALITY['Low'])
-
-        # 5. Enter/Exit game mode.
-        TestHelper.enter_game_mode(Tests.enter_game_mode)
-        general.idle_wait_frames(1)
-        TestHelper.exit_game_mode(Tests.exit_game_mode)
+        # Add a Diffuse Probe Grid since the Diffuse Global Illumination level component sets values for that component
+        diffuse_probe_grid_entity = EditorEntity.create_editor_entity(AtomComponentProperties.diffuse_probe_grid())
+        diffuse_probe_grid_entity.add_component(AtomComponentProperties.diffuse_probe_grid('shapes')[0])
+        diffuse_probe_grid_entity.add_component(AtomComponentProperties.diffuse_probe_grid())
+
+        # 4. Set Quality Level property
+        for quality in GLOBAL_ILLUMINATION_QUALITY:
+            diffuse_global_illumination_quality = (
+                f"Quality Level set to {quality}",
+                f"P1: Quality Level could not be set to {quality}")
+            diffuse_global_illumination_component.set_component_property_value(
+                AtomComponentProperties.diffuse_global_illumination('Quality Level'),
+                GLOBAL_ILLUMINATION_QUALITY[quality])
+            get_quality = diffuse_global_illumination_component.get_component_property_value(
+                AtomComponentProperties.diffuse_global_illumination('Quality Level'))
+            Report.result(diffuse_global_illumination_quality, get_quality == GLOBAL_ILLUMINATION_QUALITY[quality])
+
+            # 5. Enter/Exit game mode.
+            general.idle_wait_frames(5)
+            TestHelper.enter_game_mode(Tests.enter_game_mode)
+            general.idle_wait_frames(5)
+            TestHelper.exit_game_mode(Tests.exit_game_mode)
```

Comment:
Looks like this block of code is indented further than the rest. I'm surprised Python didn't complain/fail. Shouldn't it match the indent of the rest of the steps?

Reply:
the indentation of enter/exit game mode is intentional. it's part of the step 4 for loop I wanted the enter/exit game mode to run for each quality level since the quality impact is partly a game mode thing.

Reply:
ah makes sense then, wasn't obvious at first that it is part of the for loop but now I see it